apis/comms_api/models/commands.py

```diff
@@ -12,12 +11,3 @@ class Commands(BaseModel):
 
 class CommandsResults(BaseModel):
     results: List[Result]
-
-    @model_validator(mode='after')
```

Comment:
Had to remove this for the tests to pass, it doesn't really matter because the endpoint will be removed in the next issue related to the commands manager